__init__.py

```
@@ -28,32 +28,27 @@ def debug(*variables: Any) -> None:
-
 # FUNCTIONS -> HANDLER
 def handler(requestModel: Any, responseModel: Any, extensions: list[Any]) -> Any:
     def decorator(output: Any) -> Any:
         async def hand(socket: WebSocket) -> None:
             await socket.accept()
-            classList = await asyncio.gather(*[module.namespace().init(socket) for module in extensions])
             try:
                 while True:
                     try: payload = requestModel(**await socket.receive_json())
-                    except WebSocketDisconnect as error: raise error
+                    except WebSocketDisconnect: break
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)